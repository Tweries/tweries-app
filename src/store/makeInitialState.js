import { NEWLINE } from '../constants';
import makeTweetstorm from './makeTweetstorm';

function makeInitialState({ feature }) {
  return {
    hashtags: '',
    healthy: false,
    items: makeTweetstorm(feature)({
      hashtags: '',
      linefeed: NEWLINE,
      source: ''
    }),
    linefeed: NEWLINE,
    notification: null,
    source: '',
    userId: ''
  };
}

export default makeInitialState;
