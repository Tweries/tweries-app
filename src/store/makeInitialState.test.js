import makeInitialState from './makeInitialState';

test('to match snapshot', () => {
  expect(makeInitialState({ feature: jest.fn() })).toMatchSnapshot();
});
