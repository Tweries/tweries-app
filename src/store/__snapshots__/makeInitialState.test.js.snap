// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`to match snapshot 1`] = `
Object {
  "hashtags": "",
  "healthy": false,
  "items": Array [],
  "linefeed": "
",
  "notification": null,
  "source": "",
  "userId": "",
}
`;
